learning/tour-of-beam/frontend/lib/pages/home/screen.dart

```diff
@@ -17,16 +17,146 @@
+  const _SdkSelection();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const Text('Welcome to the Tour of Beam!'),
+        const Text(
+          'Your journey is broken down into learning modules. If you would like to save your progress and track completed modules, please sign in. \n\nPlease select the default language (you may change the language at any time):',
+        ),
+        const _SdkButtons(),
+        Image.asset(ProjectAssets.welcomeLaptop),
+      ],
+    );
+  }
+}
+
+class _SdkButtons extends StatelessWidget {
+  const _SdkButtons();
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO(nausharipov): make it responsive
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Row(
+          children: ['Java', 'Python', 'Go']
+              .map(
+                (e) => OutlinedButton(
+                  onPressed: () {
+                    // TODO(nausharipov): select the language
+                  },
+                  child: Text(e),
+                ),
+              )
+              .toList(),
+        ),
+        ElevatedButton(
+          onPressed: () {
+            // TODO(nausharipov): redirect
+          },
+          child: const Text('Start your tour'),
+        ),
+      ],
+    );
+  }
+}
+
+class _TourSummary extends StatelessWidget {
+  const _TourSummary();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: ['Core Transforms']
+          .map(
+            (e) => const _Module(),
+          )
+          .toList(),
+    );
+  }
+}
+
+class _Module extends StatelessWidget {
+  const _Module();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: const [
+        _ModuleHeader(),
+        _ModuleBody(),
+      ],
+    );
+  }
+}
+
+class _ModuleHeader extends StatelessWidget {
+  const _ModuleHeader();
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        SvgPicture.asset(ProjectAssets.welcomeProgress0),
+        const Text('Core Transforms'),
+      ],
+    );
+  }
+}
+
+class _ModuleBody extends StatelessWidget {
+  const _ModuleBody();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.only(left: 17),
+      decoration: const BoxDecoration(
+        border: Border(
+          left: BorderSide(),
+        ),
+      ),
+      child: Column(
+        children: const [
+          Text(
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam velit purus, tincidunt id velit vitae, mattis dictum velit. Nunc sit amet nunc at turpis eleifend commodo ac ut libero. Aenean rutrum rutrum nulla ut efficitur. Vestibulum pulvinar eros dictum lectus volutpat dignissim vitae quis nisi. Maecenas sem erat, elementum in euismod ut, interdum ac massa.',
+          ),
+          Divider(),
```

Comment:
These don't always work in HTML renderer. So I made this one:
https://github.com/apache/beam/pull/22477/files#diff-dc69bd863fc12cee858323b9d8e9e565fe3dd27bf927ebaecb78aec89fca2b7e

You can add it to the common playground widgets package, cleaning up the playground constants and making height a local constant.

Reply:
What cases isn't Divider shown in? I've checked it in Chrome, Safari and Firefox and it works there.

Reply:
Not sure. There is a Flutter issue: https://github.com/flutter/flutter/issues/46339
And we faced it in Playground.
It's no overhead, because this widget will be in the shared package for the two.